Filters/exclusions.txt

```diff
@@ -1,3 +1,6 @@
+! https://github.com/AdguardTeam/AdGuardSDNSFilter/issues/1780
+! Exclude domain from popup filter, leave tracking subdomains
+|getresponse.com^
```

Comment:
Few questions, because I don't understand how it works currently.

1. As far as I understand, this rule is from `AdGuard DNS Popup Hosts filter`, right?
2. If so, if it will be excluded from `AdGuard DNS filter` (I cannot find such rule in DNS filter, I see only `||ga.getresponse.com^`), will it be also excluded from `AdGuard DNS Popup Hosts filter`?
3. And where can I find `AdGuard DNS Popup Hosts filter`?

Reply:
The rule in AdGuardFilters should be changed.
It was added in this commit, but I cannot check in now :harold:

Reply:
But this rule is in `AnnoyancesFilter\Popups\sections\popups_general.txt` so it shouldn't be rather be available in `AdGuard DNS filter`?

There are also:
```adblock
||getresponse.com^$popup
||getresponse.com^$third-party
```
in French filter, but still I don't see `||getresponse.com^` in DNS filter.

Reply:
AdGuard DNS Popup Hosts filter only available in private DNS

<details>
<summary>Details</summary>

![image](https://github.com/user-attachments/assets/f4e8ddbf-08fe-4f21-a7da-4b7a9332c00c)

</details>

Reply:
> If so, if it will be excluded from AdGuard DNS filter (I cannot find such rule in DNS filter, I see only ||ga.getresponse.com^), will it be also excluded from AdGuard DNS Popup Hosts filter?

I excluded with one **|** so that tracking like `ga.getresponse.com` would bock, but the rule. 
`||getresponse.com^$dnsrewrite=ad-block.dns.adguard.com` has been excluded.
The reason this rule gets caught is because we have **$popup** enabled in the configuration - `configuration_popup_filter.json` and not enabled in `configuration.json`

```
‘inclusions": [‘/^\\\\|\\\\|[a-z0-9-.]+\\\^(\\$(popup|popup,document|document|document,popup|all))$/’],
```

You can try to build locally from my branch `yarn build` and make sure that the rule is not in `adguard_popup_filter.txt` and is in `filter.txt`.

Reply:
Thank you for your answer, I have learned something.

<hr>

>I excluded with one | so that tracking like `ga.getresponse.com` would bock, but the rule
`||getresponse.com^$dnsrewrite=ad-block.dns.adguard.com` has been excluded.

Frankly, I don't see a difference between `|getresponse.com^` and `||getresponse.com^` when I build it locally.
In both cases it's removed from `adguard_popup_filter.txt` and `||ga.getresponse.com^` is still in `filter.txt`.
If I'm not mistaken, if a rule is matched by entry added to exclusion, then it will be excluded.
So if there is `|getresponse.com^`, then it will exclude `|getresponse.com^` and `||getresponse.com^`, but if there would be `||getresponse.com^` then only rule which match `||getresponse.com^` will be excluded, and `||ga.getresponse.com^` should not be excluded by both "rules".

Reply:
Re-checked, indeed. Updated.